samtranslator/model/sam_resources.py

```diff
@@ -867,8 +868,123 @@ def to_cloudformation(self, **kwargs):
             resources.append(role)
             synthetics_canary.ExecutionRoleArn = role.get_runtime_attr("arn")
 
+        if self.CloudWatchAlarms:
```

Comment:
I thought the design doc said CanaryAlarms or CanaryMetricAlarms given that we only are supporting metrics by Canary.